torch/nn/modules/lazy.py

```diff
@@ -226,8 +228,9 @@ def has_uninitialized_params(self: _LazyProtocol):
         # This is to avoid the JIT to track this parameter and force
         # custom modules __setstate__ to add it
         params = self._parameters.values()
-        for param in itertools.chain(params):
-            if isinstance(param, (UninitializedParameter)):
+        buffers = self._buffers.values()
+        for param in itertools.chain(params, buffers):
+            if isinstance(param, UninitializedMixin):
```

Comment:
There are quite a lot of calls to this isinstance check in a couple files and the "UninitializedMixin" might be hard to discover for people.
I still think that having a specific method to check this would make things easier to read and better for future modifications.

Reply:
OK! I will add `is_lazy` function.